modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts

```diff
@@ -43,7 +43,7 @@ import { CommandEvent, ICommandEventData, DomainEvent,
 } from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
 import { randomUUID } from "crypto";
 
-
+jest.setTimeout(20000)
```

Comment:
Please put a comment here as to why this is needed :D

Reply:
done